python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -116,25 +134,31 @@ def convert_out_position(x, y, inr, c, cent):
     return xx, yy
 
 
-def convert_in_position(xyin, inr, c, cent):
+def convert_in_position(xyin, za, inr, pa, c, cent, time):
     """convert input position to those on the same coordinates as
         the WFC as-built model.
     Parameters
     ----------
-    xyin : `np.ndarray`, (N, 2)
+    xyin : `np.ndarray`, (2, N)
         Input coordinates.
         Unit is degree for sky, mm for PFI, and pixel for MCS
```

Comment:
Minor: reword to 'Input coordinates, in units of degree for sky...'

Reply:
I reworded this and similar sentences at other places.